projects/rocblas/clients/openmp-support/CMakeLists.txt

```
@@ -0,0 +1,64 @@
+# Copyright Advanced Micro Devices, Inc., or its affiliates.
+# SPDX-License-Identifier: MIT
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/clients/staging")
+
+find_package(OpenMP CONFIG)
+add_library(rocblas-omp-shim INTERFACE)
+add_library(rocblas::omp-shim ALIAS rocblas-omp-shim)
```

Nice pattern.

```suggestion
add_library(rocblas-omp-interface INTERFACE)
add_library(rocblas::omp-interface ALIAS rocblas-omp-interface)
```
Nit: "shim" feels a bit jargony here, it makes sense because it's a thin adapter layer, but "interface" feels clear and doesn't clash with the similarly named rocblas_shim, which is.. different.